packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -744,6 +746,9 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
 
     protected createEditorDecorations(resultNode: SearchInWorkspaceFileNode | undefined): EditorDecoration[] {
         const decorations: EditorDecoration[] = [];
+        const findMatchHighlightBackground = typeof this.colorRegistry.getCurrentColor('editor.findMatchHighlightBackground') !== undefined
```

Comment:
why to query twice and not simply use a result of `this.colorRegistry.getCurrentColor('editor.findMatchHighlightBackground')`?

Reply:
Thank you for your feedback, I've updated the code to only query once :)